src/modules/display.js

```diff
@@ -0,0 +1,38 @@
+import get from './get';
+import post from './post.js';
+
+const nameInput = document.querySelector('.name');
+const scoreInput = document.querySelector('.score');
+const tbody = document.querySelector('.tbody');
+const submitBtn = document.querySelector('.submit');
+const refreshbtn = document.querySelector('.refresh');
+
+const display = async () => {
+  tbody.innerHTML = '';
+  const names = await get();
+  //   console.log(names);
```

Comment:
- [ ] It would be nice if you remove the commented piece of code to make your codebase clean to follow the JavaScript best practices as suggested by the previous reviewer :+1: